Profiler/CProfilerCallback.cpp

```diff
@@ -53,37 +53,42 @@ HRESULT CProfilerCallback::InitializeImplementation(IUnknown* pICorProfilerInfoU
 		return S_OK;
 	}
 
-	log.createLogFile(config);
-	log.info("looking for configuration options in: " + config.getConfigPath());
+	// Place the attach log next to the config and profiler dll
+	std::string configPath = StringUtils::removeLastPartOfPath(config.getConfigPath());
+	attachLog.createLogFile(configPath);
+	attachLog.logAttach();
+
+	traceLog.createLogFile(config);

```

Comment:
For consistency, I think both `createLogFile` methods should take the directory path to write the log file to as an argument. Passing the entire `config` just to call `config.getTargetDir()` in the first line is unnecessary coupling anyways.